namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-historical-prisoner-dev/resources/rds.tf

```diff
@@ -1,41 +1,49 @@
-module "hmpps_hpa_rds" {
+module "hpa_rds" {
```

Comment:
change module name so that terraform just gets rid of old database and creates a new one in case there are issues with switching the engine from express to web